### DISCUSSION TITLE
Collections - What are they?

### DISCUSSION THREAD

**Discussion by Johnson90512** [advplyr/audiobookshelf#1217] [Q&A]
So I've got this setup and I love the software. I've got series built, and I've got sub series, but I haven't figured out the use case for a collection yet. How are you using them or how are they supposed to be used? I'm intrigued.

**Comment by hobesman** [comment]
FYI the discord is very active and posts there will likely get more responses. The GitHub is primarily aimed at bugs and feature requests.

But right now, I use a collection for books that I plan to listen to next. I have about 6500 books so a smaller collection is useful to gather the ones I'm most interested in right now. I haven't found a good use case beyond that, but I think others have.
